.github/workflows/go.yml

```diff
@@ -23,13 +23,13 @@ jobs:
       uses: actions/checkout@v3
 
     - name: Build for AMD64
-      run: GOOS=linux GOARCH=amd64 go build -v -a -x ./...
+      run: GOOS=linux GOARCH=amd64 go build -tags jsoniter -v -a -x ./...
```

Comment:
Ensures that gin uses `jsoniter` to parse JSON (see also https://github.com/els0r/goProbe/issues/111#issuecomment-1667397778 for more details).